app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+from shutil import copyfileobj
+
+
+def copy_file(command: str) -> None:
+    parts = command.split()
+    source_filename, destination_filename = parts[1], parts[2]
+    if (len(parts) == 3 and parts[0] == "cp"
+            and source_filename != destination_filename):
```

Comment:
You could replace the use of two `and` with `all([...])`, where you can pass three conditions in a list.